tuned/profiles/locator.py

```diff
@@ -54,10 +52,10 @@ def parse_config(self, profile_name):
 		if config_file is None:
 			return None
 		try:
-			config = ConfigParser()
+			config = ConfigParser(delimiters=('='), inline_comment_prefixes=('#'), allow_no_value=True)
```

Comment:
Why do we have in some `ConfigParser()` instantiations `allow_no_value=True` and not in others?  Could you share an example where `allow_no_value=True` is neccessary?

Reply:
Here are looser rules, so we can locate even wrongly written profiles, otherwise there would be no possibility to show it to user in list of profiles and he would need to know that if he creates profile with mistake, it won't show anywhere and he needs to manually locate it in directory.
In variables.py there is no other possibility to use None vars. I don't know if there is any real world situation when somebody would want that, but it was possible before and we didn't really check syntax of these files, so I left it compatible.